src/main/java/me/magnet/consultant/Consultant.java

```diff
@@ -356,6 +367,9 @@ public Consultant build() {
 			instanceName = Optional.ofNullable(instanceName)
 					.orElse(Optional.ofNullable(fromEnvironment("SERVICE_INSTANCE"))
 							.orElse(UUID.randomUUID().toString()));
+			Arrays.stream(Strings.nullToEmpty(fromEnvironment("SERVICE_TAGS")).split(","))
+					.filter((t) -> t != null && !t.isEmpty())
```

Comment:
when I have a lot of streams that need negation I usually just copy-paste these lines:
```
public static Function<T, Boolean> not(Function<T, Boolean> function) {
   return (input) -> !function.apply(input);
}
````

Quick fix, and let's me do `filter(not(Strings::isNullOrEmpty))`. But I'd say that might be a bit much for this PR :wink: